protos/netstate/netstate.proto

```diff
@@ -31,22 +87,22 @@ message GetRequest {
 
 // ListRequest is a request message for the List rpc call
 message ListRequest {
-  bytes bucket = 1;
+  bytes starting_path_key = 1; // the Path key in the bucket to start listing
+  int64 limit = 2; // how many keys to list
 }
 
 // PutResponse is a response message for the Put rpc call
 message PutResponse {
-  string confirmation = 1;
 }
 
 // GetResponse is a response message for the Get rpc call
 message GetResponse {
-  bytes smallValue = 1;
+  bytes pointer = 1; // this is a Pointer type marshalled into bytes
```

Comment:
probably makes sense to have this match the PutRequest. I dunno if that means PutRequest takes a bytes or GetResponse returns a Pointer?

this will probably change slightly once we get to signatures so it doesn't matter much right now

Reply:
I have `bytes pointer` returned in the Get Response so that they can be unmarshalled after the transfer (I thought this would make it smaller and more convenient to transfer?), but I'll bookmark this for the future, thanks!

Reply:
what gets transferred across the network is actually almost identical and i would bet the exact same amount of bytes. the reason is because the get response is already a protocol buffer, it will get marshaled before being sent anyway, so whether or not you do the marshaling for it, the pointer will get marshaled anyways

Reply:
I didn't realize this, good to know! 👌